zebrad/tests/acceptance.rs

```diff
@@ -860,6 +864,10 @@ fn create_cached_database_height(network: Network, height: Height) -> Result<()>
     let network = format!("network: {},", network);
     child.expect_stdout_line_matches(&network)?;
     child.expect_stdout_line_matches(STOP_AT_HEIGHT_REGEX)?;
+
+    child.expect_stdout_line_matches("starting legacy chain check")?;
+    child.expect_stdout_line_matches("no legacy chain found")?;
```

Comment:
Bug: the legacy chain check happens when the database is opened. Then after the chain has synced for a while, we stop at the configured height.
```suggestion

    child.expect_stdout_line_matches("starting legacy chain check")?;
    child.expect_stdout_line_matches("no legacy chain found")?;

    child.expect_stdout_line_matches(STOP_AT_HEIGHT_REGEX)?;
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/2366/commits/e3208f42579ba586705a35108c47c0b2e120c0c6